### ISSUE TITLE
standard_init_linux.go:219: exec user process caused: no such file or directory

### ISSUE THREAD

**Issue by alelav79** [kylemanna/docker-openvpn#663]
Hi,

i have this error 

# docker run -v C:\path\vpn-data:/etc/openvpn --rm myownvpn ovpn_genconfig -u udp://192.168.3.54:3000
standard_init_linux.go:219: exec user process caused: no such file or directory

what is the problem and solution for ?

Thanks

**Comment by mazzy89** [comment]
you are running in a wrong arch. your container that you are running is not supported. my gut feeling is that you are running on arm or something like that and the container you have is for amd64.

**Comment by alelav79** [comment]
Sorry, 

my Arch is 

λ wmic OS get OSArchitecture
OSArchitecture
64 bit

**Comment by mazzy89** [comment]
run
```
uname -a
```

**Comment by alelav79** [comment]
MSYS_NT-10.0-19042 MYPC 3.1.6-340.x86_64 2020-07-09 14:33 UTC x86_64 Msys

**Comment by mazzy89** [comment]
ok your arch is `x86_64`. are you running the docker upstream image? can you post which image are you running?

**Comment by mazzy89** [comment]
oh wait but you are running some Windows OS right? I see there `C:/`.

**Comment by alelav79** [comment]
Yes, its Windows 10 .

**Comment by mazzy89** [comment]
oh, that explains a lot. then you need to build your container to support Windows. According the maintainer it has not been tested on Windows machine https://github.com/kylemanna/docker-openvpn#originally-tested-on

**Comment by alelav79** [comment]
I need Windows support.

Is there anything I can do ?

Thanks.

**Comment by mazzy89** [comment]
Yeah you would need to fork this project and build by yourself making the needed changes. Or ask to the maintainer if he is so kind to distribute this feature. To be honest never used in my life Docker containers for Windows so I cannot help here.
